warehouse/templates/base.html

```diff
@@ -161,17 +162,29 @@ <h3>About PyPI</h3>
           {% if request.registry.settings.get("statuspage.url") %}
           <li><a href="https://status.python.org/">Status: <span data-statuspage-domain="{{ request.registry.settings['statuspage.url'] }}">All Systems Operational</span></a></li>
           {% endif %}
-          <li><a href="{{ request.route_path('security') }}">Security Policy</a></li>
+          <li><a href="https://www.python.org/dev/peps/pep-0541/" class="tooltipped tooltipped-e" aria-label="Package Index Name Retention" data-original-label="Package Index Name Retention">PEP 541</a></li>
```

Comment:
I think this link text should be the title of the PEP rather than just the tooltip (otherwise folks might not realize that "PEP 541" is what they want when looking for how to resolve naming issues.)